lib/bolt/transport/winrm.rb

```diff
@@ -31,7 +31,8 @@ class WinRM < Base
         "ssl"             => "When true, Bolt uses secure https connections for WinRM.",
         "ssl-verify"      => "When true, verifies the targets certificate matches the cacert.",
         "tmpdir"          => "The directory to upload and execute temporary files on the target.",
-        "user"            => "Login user. **Required unless using Kerberos.**"
+        "user"            => "Login user. **Required unless using Kerberos.**",
+        "basic-auth-only" => "Force basic authentication."
```

Comment:
`basic-auth-only` can be added to the `Bolt::Config::Transport::WinRM::OPTIONS` hash with the `type` set to `TrueClass`: https://github.com/puppetlabs/bolt/blob/master/lib/bolt/config/transport/winrm.rb#L10-L50